app/models/miq_worker.rb

```diff
@@ -339,26 +316,6 @@ def queue_name
     end
   end
 
-  def start_runner
-    if ENV['MIQ_SPAWN_WORKERS'] || !Process.respond_to?(:fork)
-      start_runner_via_spawn
-    else
-      start_runner_via_fork
-    end
-  end
```

Comment:
Does this mess up @carbonin 's containers fork?

Reply:
Yeah, but don't worry about that. The rebase is going to be a disaster either way :laughing:

Reply:
_womp womp_